File: app/src/main/java/org/dhis2/data/database/SqlBriteLogger.java
package org.dhis2.data.database;


import com.squareup.sqlbrite2.SqlBrite;

import timber.log.Timber;

class SqlBriteLogger implements SqlBrite.Logger {
    private static final String TAG = SqlBriteLogger.class.getSimpleName();

    @Override
    public void log(String message) {
        Timber.tag(TAG);
        Timber.d(message);
    }
}

File: app/src/main/java/org/dhis2/data/database/SqlBriteLogger.kt
package org.dhis2.data.database


import com.squareup.sqlbrite2.SqlBrite

import timber.log.Timber

internal class SqlBriteLogger : SqlBrite.Logger {

    override fun log(message: String) {
        Timber.tag(TAG)
        Timber.d(message)
    }

    companion object {
        private val TAG = SqlBriteLogger::class.java.simpleName
    }
}
